### DISCUSSION TITLE
0.5 release

### DISCUSSION THREAD

**Discussion by amitu** [fastn-stack/fastn#1446] [General]
In [fastn 0.4](https://github.com/orgs/fastn-stack/discussions/1079) we aimed for fastn as a react replacement use case, added dynamic dom, improved page performance etc. We are focusing on fastn for content centric sites use case in 0.5 release.

- Core Features
  - [ ] [fastn tutor](https://github.com/orgs/fastn-stack/discussions/1414): @amitu 
  - [x] [configuration unification](https://github.com/orgs/fastn-stack/discussions/1451) @siddhantk232 
  - [x] proper frontend form handling, which includes [ftd.json](https://github.com/orgs/fastn-stack/discussions/1035) @Heulitig 
- Excel / reporting / analytics use case
  - [x] [google sheet query](https://github.com/orgs/fastn-stack/discussions/1450) @harshdoesdev 
  - [ ] [chart.js integration](https://github.com/orgs/fastn-stack/discussions/1427) @harshdoesdev 
- language features @Arpita-Jaiswal 
  - [ ] motion (scroll animation) effects (we want pages created using fastn to be fancy)
  - [ ] view transitions (fancy)
  - [ ] page size reduction: common.js (dependencies used in index.ftd file), and doc-name.js (for each document we will do a dependency delta from common.js, and that documents js, put them in doc-name.js).
  - [ ] service worker navigation capture (allows us to use view transition across page change, and generally improve performance by not unloading-loading new page).
  - [ ] `ftd.scene`
  - [ ] [hover style](https://github.com/orgs/fastn-stack/discussions/975)
- website features
  - [ ] mixed privacy (authentication etc) @siddhantk232 
  - [ ] notification @Arpita-Jaiswal 
  - [ ] subscription (PostgreSQL backed?) @harshdoesdev  
  - [ ] search, or better chat with your docs @harshdoesdev

**Comment by harshdoesdev** [comment]
Please take a look at [`importmap`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap), it can help us in in improving code readability and simplify how we are resolving our js dependencies.

**Comment by amitu** [comment]
This is quite outdated now, closing it in favour of [another 0.5 release tracker](https://github.com/orgs/fastn-stack/discussions/2041).
